api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -47,6 +47,13 @@ message UIOnboardSetCredentialSubmitEvent {
   string username = 1;
 }
 
+// UIOnboardQuestionnaireSubmitEvent is a UI event sent during registration when
+// user submit their onboarding questionnaire.
+message UIOnboardQuestionnaireSubmitEvent {
+  // The event username. Not extracted from identity as this is pre-registration.
```

Comment:
"The event username" is a bit confusing to me (as an outsider without much context).